os/include/string.h

```diff
@@ -201,6 +201,13 @@ FAR char *strcpy(char *dest, FAR const char *src);
  * @since TizenRT v1.0
  */
 FAR char *stpcpy(FAR char *dest, FAR const char *src);
+/**
+ * @brief copy fixed length string, returning a pointer to the array end
+ * @details @b #include <string.h> \n
+ * POSIX API (refer to : http://pubs.opengroup.org/onlinepubs/9699919799/)
+ * @since TizenRT v1.0
+ */
+FAR char  *stpncpy(FAR char *, FAR const char *, size_t);
```

Comment:
Could you fill name of arguments? That is an our coding rule.

Reply:
@sunghan-chang 
There are a variety of argument names used for cpy API's
Is the below declaration fine ??
FAR char  *stpncpy(FAR char *s1, FAR const char *s2, size_t n);